database/migrations/2025_05_02_133959_filter_empty_socialite_configs.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use App\Facades\LibrenmsConfig;
+use Illuminate\Database\Migrations\Migration;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        $configs = LibrenmsConfig::get('auth.socialite.configs', []);
```

Comment:
I don't think this should use LibrenmsConfig, but work on the DB directly instead.